flink-tests/src/test/java/org/apache/flink/test/checkpointing/AutoRescalingITCase.java

```diff
@@ -163,6 +163,8 @@ public void setup() throws Exception {
                     NettyShuffleEnvironmentOptions.NETWORK_BUFFERS_PER_CHANNEL, buffersPerChannel);
 
             config.set(JobManagerOptions.SCHEDULER, JobManagerOptions.SchedulerType.Adaptive);
+            // Disable the scaling cooldown to speed up the test
+            config.set(JobManagerOptions.SCHEDULER_SCALING_INTERVAL_MIN, Duration.ofMillis(0));
```

Comment:
Disable the scaling cooldown will increase the hang possibility, so I didn't merge it in 1.19 before.

Reply:
Why does it increase the hang possibility? Why do we merge it then now? And why did we merge it into `master`? :thinking:

Reply:
Hi @XComp , thanks for your quick review.

Please see this comment. https://github.com/apache/flink/pull/24340/files#r1495400609